src/Symfony/Component/Routing/Route.php

```diff
@@ -124,8 +124,6 @@ public function getPath()
     /**
      * Sets the pattern for the path.
      *
-     * This method implements a fluent interface.
```

Comment:
there are more of those sentences in this file that you could remove for consistency